_includes/v21.2/prod-deployment/healthy-lsm.md

```diff
@@ -0,0 +1 @@
+**Expected values for a healthy cluster**: The number of L0 files should be fewer than 1000. High values indicate heavy write load that is causing accumulation of files in level 0. These files are not being compacted quickly enough to lower levels, resulting in a misshapen LSM.
```

Comment:
"Fewer than 1000" is based on the current threshold at which admission control takes over. Not sure if we have better guidance on this yet.

Reply:
Ballpark is correct. Emphasize order of magnitude. Files in the hundreds to low thousands is OK for bigger servers.

High thousands are not good.

There is additional nuance here relating to vCPU.